test/tests/multiapps/picard_multilevel/2level_picard/mutilevel_app.i

```diff
@@ -69,6 +69,7 @@
   fixed_point_rel_tol = 1E-3
   fixed_point_abs_tol = 1.0e-05
   fixed_point_max_its = 2
+  accept_on_max_fixed_point_iteration = true
```

Comment:
this one is ok. This test should have allowed for more iterations but with this option the test still meets its purpose